templates/questions/index.html.ep

```diff
@@ -5,6 +5,9 @@
 	<script src="/js/scripts.js"></script>
 	<link rel="stylesheet" media="screen and (max-device-width: 800px)" href="/css/style.css">
 	<link rel="stylesheet" media="screen and (min-device-width: 800px)" href="/css/desktop.css">
+	<% use Gravatar::URL; %>
```

Comment:
This works, but doesn't seem like the best place for it.  I'd load your model in Ama.pm.  I don't honestly have a good reason for that.

Something else that I would do, we talked about "silly" one-line libraries...  I'd create a Mojolicious::Plugin::Gravatar that's a loader for this library.  It would allow you to:

```
$self->plugin('gravatar');
```

And then use it later with

```
$self->gravatar->gravatar_url
```

You could probably even swing it so that it could look even better

```
$self->gravatar->url
```

Make sense?  Any questions?